File: bulk/java/Xss01.java
// CWE-79: XSS in JSP via out.println
// In a real JSP: <%= request.getParameter("name") %>
public class Xss01 {
    public String renderPage(javax.servlet.http.HttpServletRequest req) {
        String name = req.getParameter("name");
        return "<html><body>Hello " + name + "</body></html>"; // reflected XSS
    }
}

File: bulk/csharp/Xss01.cs
// CWE-79: XSS via HtmlString in Razor
using Microsoft.AspNetCore.Html;
public class Xss01 {
    public HtmlString Render(string userInput) {
        return new HtmlString("<div>" + userInput + "</div>"); // unescaped
    }
}
